src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1819,4 +1820,35 @@ namespace NServiceBus.Transports.Msmq.Config
     public class MsmqSettings
     {
     }
-}
\ No newline at end of file
+}
+
+public static class ConfigureHandlerSettings
+{
+    [ObsoleteEx(
+         RemoveInVersion = "7.0",
+         TreatAsErrorFromVersion = "6.0",
+         Message = "Setting property values explicitly is no longer supported via the builtin container in NServiceBus. Property injection is still enabled by default. Use the `.ConfigureComponent(b=> new MyMessageHandler(){ MyProperty = X})` if you want full controll over property values. Other option is to use one of the supported containers and configure property injection using their native configuration API's.")]
```

Comment:
> Use the `.ConfigureComponent(b=> new MyMessageHandler(){ MyProperty = X})` if you want full controll over property values.

use the [...] **API** if you want [...] ?

> Other option is to use one of the supported containers and configure property injection using their native configuration API's.

The other option? Otherwise? Alternatively??